consumer/pdata/spanid.go

```diff
@@ -40,6 +40,12 @@ func (t SpanID) HexString() string {
 }
 
 // IsValid returns true if id contains at leas one non-zero byte.
+// Deprecated: use !IsEmpty() instead.
 func (t SpanID) IsValid() bool {
 	return data.SpanID(t).IsValid()
 }
+
+// IsEmpty returns true if id doesn't contain at least one non-zero byte.
+func (t SpanID) IsEmpty() bool {
+	return !data.SpanID(t).IsValid()
```

Comment:
Should we also change `data.SpanID` to have `IsEmpty` instead of `IsValid`?

Reply:
will do it in a followup PR.